### PR TITLE
Faq update

### PR DESCRIPTION
**Pull request by Rspigler**
Mostly clarity, run-off sentence, and grammar stuff.

Some spelling.

Tried to keep all points in tact and in order.

Probably only controversial change is change from WIF key description to `hdseed`.  Because IMO I didn't think that made sense unless we described it further since WIF is single key (that we set WIF as `hdseed`; which I thought was confusing to put in the FAQ).  So technically we are backing up `hdseed` (just changing it first)? Iffy, but also what we are trying to refactor to here: https://github.com/JWWeatherman/yeticold/issues/106 so it'll be a more accurate description soon.

Open to discussion.

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -11,52 +11,51 @@ Level 3
...
 https://join.slack.com/t/yeticold/shared_invite/zt-hisfxrra-BZzrYCDnqFv6whxVn~6FQQ
 
 ### How much knowledge of Linux is required?
-None. Installing Ubuntu is the only challenging task and it is no more difficult than installing Windows or Mac OS. Even if you have never installed an OS before it is not difficult and you can get support from our slack channel at:
+None. Installing Ubuntu is the only challenging task and it is no more difficult than installing Windows or Mac OS. Even if you have never installed an OS before it is not difficult. Our guide offers instructions, and the linked videos offer a more comprehensive walkthrough. You can also get support from our slack channel at:
 
 https://join.slack.com/t/yeticold/shared_invite/zt-hisfxrra-BZzrYCDnqFv6whxVn~6FQQ
 
-You can also hire a local computer shop to install Ubuntu for you for about $30 and as long as they don't suspect you will use this laptop for bitcoin it is a reasonably safe option.
+You can also hire a local computer shop to install Ubuntu for you for about $30 and as long as they don't suspect you will use this laptop for bitcoin it is a reasonably safe option for lower levels.
```
Comment (JWWeatherman) [inline FAQ.md:23]: I think this is a reasonably safe option even for Level 3.

**Review by JWWeatherman — APPROVED** [review]
Looks good. I would like that one line corrected to include L3, but I'm going to merge this in as is since it is easy enough to revert if we agree later.

**Comment by JWWeatherman** [comment]
@willweatherman can you double check this and merge it in once you are certain it is safe. Good for us to be in the habit even on text changes of multiple checks.

File: FAQ.md (hunk 1 above)
Comment (Rspigler) [reply FAQ.md]: Fixed.

**Pull request merged**